cmd/influxd/run/server_test.go

```diff
@@ -7356,3 +7356,38 @@ func TestServer_Query_Sample_Wildcard(t *testing.T) {
 		}
 	}
 }
+
+// Validate that nested aggregates don't panic
+func TestServer_NestedAggregateWithMathPanics(t *testing.T) {
+	t.Parallel()
+	s := OpenServer(NewConfig())
+	defer s.Close()
+
+	if err := s.CreateDatabaseAndRetentionPolicy("db0", newRetentionPolicySpec("rp0", 1, 0), true); err != nil {
+		t.Fatal(err)
+	}
+
+	writes := elapsedTimeWriteData
+
+	test := NewTest("db0", "rp0")
+	test.writes = Writes{
+		&Write{data: strings.Join(writes, "\n")},
+	}
+
+	test.addQueries([]*Query{
+		&Query{
+			name:    "dividing by elapsed count should not panic",
+			params:  url.Values{"db": []string{"db0"}},
+			command: `SELECT sum(count::float) / elapsed(sum(count::float), 1m) FROM jvm_threads WHERE time > '2017-01-25T06:40:41.663090514Z' GROUP BY time(1m) fill(previous)`,
```

Comment:
The expected output is so large it becomes chunked and the test suite isn't set up for that.  I would have to put some work into the test suite to see that happen.